### DISCUSSION TITLE
Is there a way to set a max width of a table column/limit its max size?

### DISCUSSION THREAD

**Discussion by evidentsnow** [palexdev/MaterialFX#91] [Archived]
It's very hard to fit all the details of a table into a small window because it seems impossible to limit the width of a column.

For example, this statement does not work (the type is an MFXTableColumn)

colID.setMaxWidth(10);

Is there another way?

**Comment by palexdev** [comment]
Yes there should be
Columns are just controls, setting they're max width should work
